src/app/[locale]/(default)/about/LabOps/page.tsx

```diff
@@ -0,0 +1,124 @@
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from '@/components/ui/Accordion';
+import { Card } from '@/components/ui/Card';
+import { useTranslations } from 'next-intl';
+import { unstable_setRequestLocale } from 'next-intl/server';
+import Image from 'next/image';
+
+import * as React from 'react';
```

Comment:
React is already imported everywhere when using Next.js you only need to import from it when you need to use hooks